### PR TITLE
updated state and props for icon Parley setting

### PR DESCRIPTION
**Pull request by boukeTracebuzz**
This PR adds a new setting, called `runOptions.icon`, which enables developers to use a custom icon for the chat button.
This setting is the same as in the v1 version of the chat (documented [here](https://developers.parley.nu/docs/settings#runoptions))

Here is an example of different icons used. The first icon is the default one and that is an SVG which is bundled with the chat. The second and third icons are loaded as a `data:image/png;base64,xxxxx` string in the ``'s `src` attribute (anything that is valid in the `src` attribute can be used as a value for this setting)
gif

### CODE REVIEW

File: cypress/fixtures/custom-chat-icon2.png
[the remark this answers is not in the document]
Comment (Gerben-T) [reply cypress/fixtures/custom-chat-icon2.png:1]: Hmm denk dat er iets mis was gegaan binnen wsl, heb ze nu gefixed als het goed is

[the remark this answers is not in the document]
Comment (Gerben-T) [reply cypress/fixtures/custom-chat-icon2.png:1]: Fixed in cf0b11ba42706f4a704eb11a27467eec44d51c1d en 47b1c9c90cbd8c441e9283759883f49855fe7666

File: cypress/fixtures/custom-chat-icon1.png
[the remark this answers is not in the document]
Comment (Gerben-T) [reply cypress/fixtures/custom-chat-icon1.png:1]: Fixed in cf0b11ba42706f4a704eb11a27467eec44d51c1d en 47b1c9c90cbd8c441e9283759883f49855fe7666

File: src/UI/Launcher.module.css [hunk 1]
```diff
@@ -20,3 +20,8 @@
 	text-align: center;
 	transform: translateZ(0);
 }
+
+.launcher img {
+	width: 80%;
```
[the remark this answers is not in the document]
Comment (Gerben-T) [reply src/UI/Launcher.module.css]: Fixed in ce0d18fd1deedcedb3d8e5fab660469079d135fc

File: src/UI/Launcher.jsx [hunk 2]
```diff
@@ -30,6 +31,9 @@ class Launcher extends Component {
 	}
 }
 
-Launcher.propTypes = {onClick: PropTypes.func};
+Launcher.propTypes = {
+	icon: PropTypes.string,
```
Comment (gydotitulaer) [inline src/UI/Launcher.jsx:35]: als icon `undefined` is, is dit geen string toch? Faalt dan deze propType niet?

**Review by gydotitulaer — DISMISSED** [review]

**Review by gydotitulaer — APPROVED** [review]

**Pull request merged**